src/main/java/com/minecolonies/coremod/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -348,4 +350,61 @@ protected int getActionsDoneUntilDumping()
     {
         return ACTIONS_UNTIL_DUMP;
     }
+
+    @Override
+    protected void sendCompletionMessage(final WorkOrderBuildDecoration wo)
+    {
+        super.sendCompletionMessage(wo);
+
+        final String structureName = job.getBlueprint().getName();
```

Comment:
why are you manually using the blueprints name instead of using the workorders display name?

Reply:
Good question